oneflow/core/boxing/symmetric_nd_sbp_boxing.cpp

```diff
@@ -60,6 +71,9 @@ Maybe<one::Tensor> Apply1DBoxing(const std::shared_ptr<one::Tensor>& input,
   const auto& boxing_interpreter =
       JUST(Global<EagerBoxingInterpreterManager>::Get()->GetEagerBoxingInterpreter(
           in_nd_sbp, out_nd_sbp, in_parallel_desc, out_parallel_desc, *input->shape()));
+  const auto& eager_boxing_logger = CachedEagerBoxingLogger();
+  eager_boxing_logger->Log(*JUST(boxing_interpreter->boxing_interpreter_status()),
+                           "\t\tInternal boxing of symmetric-nd-sbp-to-nd-sbp, " /* prefix */);
```

Comment:
```suggestion
  eager_boxing_logger->Log(*JUST(boxing_interpreter->boxing_interpreter_status()),
                           /*prefix=*/"\t\tInternal boxing of symmetric-nd-sbp-to-nd-sbp, ");
```

这里习惯这么写，类似 key-value 的传参形式

Reply:
已修改